compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1075,6 +1075,12 @@ TR::Register *OMR::X86::TreeEvaluator::integerStoreEvaluator(TR::Node *node, TR:
    if (instr && node->getOpCode().isIndirect())
       cg->setImplicitExceptionPoint(instr);
 
+   if (node->getSymbol()->isUnsafeShadowSymbol())
+      {
+      instr->setNeedsGCMap(0xFF00FFFF);
```

Comment:
I understand that part, but the `0xFF00FFFF` mask results in the GC map containing registers, which unless I am missing something I suspect you don't really want in this case.